src/controllers/activityController.js

```diff
@@ -0,0 +1,30 @@
+const Activity = require("../models/Activity");
+
+// @desc    Endpoint for fetching all activities
+// @route   GET /api/v1/activity
+// @access  public
+const getAllActivities = async (req, res) => {
+  const response = await Activity.find();
+
+  return res.json({ count: response.length, activites: response });
```

Comment:
Heads up on a small typo here -- `activites` should be `activities`.

Reply:
Noted, will be modified on the next commit.